synapse/tests/test_lib_aha.py

```diff
@@ -582,12 +582,18 @@ async def test_lib_aha_provision(self):
                     with self.getTestDir() as syndir:
                         with mock.patch('synapse.common.syndir', syndir):
```

Comment:
should be able to use `getTestSynDir` here ?

Reply:
updated